src/vsr/journal.zig

```diff
@@ -1460,6 +1512,8 @@ pub fn Journal(comptime Replica: type, comptime Storage: type) type {
         }
 
         /// `write_prepare` uses `write_sectors` to prevent concurrent disk writes.
+        // TODO To guard against torn writes, don't write simultaneously to all redundant header
+        // sectors. (This is mostly a risk for single-replica clusters very small WALs).
```

Comment:
```suggestion
        // sectors. (This is mostly a risk for single-replica clusters with small WALs).
```

Reply:
I like how you say "single-replica clusters". That's exactly it. Everything is a batch or a cluster.